lib/reporters/tap.js

```diff
@@ -118,6 +119,15 @@ class TapReporter {
 		}) + os.EOL);
 	}
 
+	writeLogs(evt) {
+		if (evt.logs) {
```

Comment:
Looking at the verbose reporter, `hook-finished` always has at least an empty array for `evt.logs` so you shouldn't need this check.